src/ispc.cpp

```diff
@@ -722,6 +748,13 @@ class AllCPUs {
         compat[CPU_AppleA17] = Set(CPU_AppleA17, CPU_None);
 #endif
 
+#ifdef ISPC_RISCV_ENABLED
+        compat[CPU_Generic_RV64GCV] = Set(CPU_Generic_RV64GCV, CPU_None);
+#if ISPC_LLVM_VERSION >= ISPC_LLVM_19_0
+        compat[CPU_SpacemiT_X60] = Set(CPU_SpacemiT_X60, CPU_Generic_RV64GCV, CPU_None);
```

Comment:
Not sure if they are compatible. Maybe follow the same logic as for Cortex devices?